mlx-swift-examples.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -83,5 +65,5 @@
       }
     }
   ],
-  "version" : 2
+  "version" : 3
```

Comment:
This update has caused problems for people in the past -- can you remove this line?

Reply:
Done.